CMSIS/Core_AArch64/Include/cmsis_compiler.h

```diff
@@ -0,0 +1,245 @@
+/**************************************************************************//**
+ * @file     cmsis_compiler.h
```

Comment:
Hi,

Did you consider adding the support inside the existing Core_A directory? (to reduce duplication)

Reply:
Well, yes I've considered to use Core_A directory at the very beginning. But then I've ran into the problem that I have to pollute existing headers by AArch64 routine implementations.